subscription-manager.spec

```diff
@@ -178,6 +191,17 @@ Requires: librsvg2
 %description -n subscription-manager-firstboot
```

Comment:
Should the firstboot stuff be disabled if use_initial_setup is true? It looks like it might build it as well on EL7.